openerp/openupgrade/doc/source/development.rst

```diff
@@ -1,7 +1,43 @@
+=========================================================
 Migration script development
-++++++++++++++++++++++++++++
+=========================================================
 
 .. toctree::
    :maxdepth: 2
 
    devfaq
+
+Migration Scrips consist of two (or more) main files in every module's migration directory located in the target source code:
+
+* pre-migration.py (or pre-anythingyouwant.py)
+* post-migration.py (or post-anythingyouwant.py)
+
+What they do is derived from their names, you should check the API Documentation_ of the openupgradelib_ for details of library methods that are readily available for your migration task.
+
+The general purpose of those scripts is to manipulate directly the database entries of the source database in order to change them in a way so that they are compatible with the target database source code. To make that happen smoothly, is your task.
+When you run a migration_, basically Odoo's *internal* updating mechanism will be used (OpenUpgrade is providing sugar code for the migration task only). 
+Expert Note: Unlike a plain Odoo upgrade, OpenUpgrade only commits after each individual module upgrade. If a migration script fails, the transaction in which that module migration is run will be rolled back. The migration process can then be restarted by running `odoo.py` only with the `-d` argument for the database (i.e. without `--update all`). However, there are some technical subtleties that prevent a continued migration without manual intervention in the database.
+
+General recommendations about 'pre-migration.py' and 'post-migration.py'
+------------------------------------------------------------------------
+TODO
+
+Special Case: Module Renames and Module Merges
+----------------------------------------------
+In a normal case, you should be able to include your module name changes into the apriori.py_ which is automatically loaded at this_ point. However, in the case you only want to migrate a specific module, things are slightly more complicated (doc improvements via PR welcome):
```

Comment:
As is evident in this part of the change, this is not covered by the OpenUpgrade distribution. Instead of getting out an OpenUpgrade branch and perform this elaborate instruction, I'd rather that people fire the necessary queries manually as described in the second part of https://gist.github.com/antespi/38978614e522b1a99563. You can leave this out or refer to this gist in a footnote if you want.